src/generated/graphql.tsx

```diff
@@ -3134,7 +3147,7 @@ export type ResultsPartsFragment = (
 
 export type ResultsPartyPartsFragment = (
   { __typename?: 'ResultsParty' }
-  & Pick<ResultsParty, 'id' | 'name' | 'logoUrl' | 'percentAgreement' | 'country'>
+  & Pick<ResultsParty, 'id' | 'name' | 'logoUrl' | 'percentAgreement'>
```

Comment:
Looks like this will break the country select option in result parties view.
Might be my fault because I added the country property manually since I am unsure how this file is generated.